README.md

```diff
@@ -48,7 +48,11 @@ var childWalletSecret = lib.derivePublic(parentWalletPublicKey, 1, 1)
 * `Buffer toPublic(Buffer privateKey)`
 * `Buffer decodePaperWalletMnemonic(string paperWalletMnemonic)`
 * `Buffer xpubToHdPassphrase(Buffer xpub)`
-* `string packAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `string deriveBootstrapAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
```

Comment:
I see the functions return the addresses as Buffers which is fine given that there are multiple ways to encode the address for the user to see it (bech32 vs base58)

So similarly as base58 is already exported, I would export a helper function to encode buffers to bech32.

Also, to keep consistency, I would change the api of packAddress/unpackAddress to use buffers instead of base58 encoded addresses